sphinx/writers/latex.py

```diff
@@ -1378,19 +1360,25 @@ def depart_hlistcol(self, node):
         pass
 
     def latex_image_length(self, width_str):
-        match = re.match('(\d*\.?\d*)\s*(\S*)', width_str)
+        return self.latex_dim_to_latexdim(width_str)
+
+    def latex_dim_to_latexdim(self, length_str):
```

Comment:
This means conversion from `latex_dim` to `latexdim`?
I prefer `convert_to_latexdim` or `rstdim_to_latexdim`.